app/controllers/users/emails_controller.rb

```diff
@@ -17,6 +17,7 @@ def add
       if result.success?
         process_successful_creation
       else
+        flash.now[:error] = t('email_addresses.add.duplicate') if duplicate_email?
```

Comment:
Wouldn't the error automatically show up if we added a validation to the form and stuck this message on the `emails` key in the errors hash?

Reply:
That prevents us from having to add flashes to the controllers for error cases and means the error automatically gets picked up by analytics.

Reply:
It would but I simply copied the format of RegisterUserEmailForm which also didn't have validation w/an email key